CHANGELOG.md

```diff
@@ -1,28 +1,37 @@
 # Changelog
+## 2.12.0
+
+- [BoxCollaboration.create](https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxCollaboration.java#L60) now takes a Boolean notify parameter to when users or groups are collaborated onto a Box Item.
```

Comment:
The method actually already had the param, we just fixed it so it works now.  We should just link to the method docs here, since the fix itself doesn't really have useful information